lang/tests/src/test/scala/com/wavesplatform/lang/v1/estimator/ScriptEstimatorV3Test.scala

```diff
@@ -114,13 +114,16 @@ class ScriptEstimatorV3Test extends ScriptEstimatorTestBase(ScriptEstimatorV3) {
     estimate(functionCosts(V3), compile(script)) shouldBe Right(2) /* ref eval and field access */
   }
 
-  property("groth16Verify_1inputs") {
+  property("groth16Verify_Ninputs") {
     implicit val version : StdLibVersion = V4
-    estimate(functionCosts(V4), compile("groth16Verify_1inputs(base64'ZGdnZHMK',base64'ZGdnZHMK',base64'ZGdnZHMK')")) shouldBe Right(1903)
+    val c = Array(1200, 1300, 1400, 1500, 1600, 1700, 1800, 1900, 2000, 2100, 2200, 2300, 2400, 2500, 2600)
```

Comment:
Do we have the same test for other limited functions? Hashes, sigVerify, rsaVerify